### ISSUE TITLE
loss 降不下来，检测结果很差，不知道什么原因，请问需要手动resize 训练和检测图片吗

### ISSUE THREAD

**Issue by 2018newyangyu** [WZMIAOMIAO/deep-learning-for-image-processing#124]
**System information**
* Have I written custom code:
* OS Platform(e.g., window10 or Linux Ubuntu 16.04):
* Python version:
* Deep learning framework and version(e.g., Tensorflow2.1 or Pytorch1.3):
* Use GPU or not:
* CUDA/cuDNN version(if you use GPU):
* The network you trained(e.g., Resnet34 network):

**Describe the current behavior**

**Error info / logs**

**Comment by WZMIAOMIAO** [comment]
用的什么代码，数据集是自己的吗？

**Comment by 2018newyangyu** [comment]
YOLOV3  嗯嗯用的自己的数据集

**Comment by WZMIAOMIAO** [comment]
- 你是严格按照我视频里说的一步步做的吗？
- 自己的数据集有在别的目标检测网络上训练过吗？效果怎么样？

**Comment by WZMIAOMIAO** [comment]
还有，你训练的时候有把```--freeze-layers```参数设置为```False```吗？

**Comment by 2018newyangyu** [comment]
嗯嗯我搞错了






------------------&nbsp;原始邮件&nbsp;------------------
发件人: "WuZhe"<notifications@github.com&gt;; 
发送时间: 2021年1月19日(星期二) 中午11:16
收件人: "WZMIAOMIAO/deep-learning-for-image-processing"<deep-learning-for-image-processing@noreply.github.com&gt;; 
抄送: "杨煜"<136240171@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [WZMIAOMIAO/deep-learning-for-image-processing] loss 降不下来，检测结果很差，不知道什么原因，请问需要手动resize 训练和检测图片吗 (#124)





 
还有，你训练的时候有把--freeze-layers参数设置为False吗？
